be/src/util/json_converter.cpp

```diff
@@ -56,116 +56,118 @@ class SimdJsonConverter {
     }
 
 private:
-    static Status convert(SimdJsonValue value, std::string_view field_name, vpack::Builder* builder) {
+    static Status convert(SimdJsonValue value, std::string_view field_name, bool has_name, vpack::Builder* builder) {
         switch (value.type()) {
         case so::json_type::array: {
-            convert(value.get_array().value(), field_name, builder);
+            convert(value.get_array().value(), field_name, has_name, builder);
             break;
         }
         case so::json_type::object: {
-            convert(value.get_object().value(), field_name, builder);
+            convert(value.get_object().value(), field_name, has_name, builder);
             break;
         }
         case so::json_type::number: {
-            convert(value.get_number().value(), field_name, builder);
+            convert(value.get_number().value(), field_name, has_name, builder);
             break;
         }
         case so::json_type::string: {
-            convert(value.get_string().value(), field_name, builder);
+            convert(value.get_string().value(), field_name, has_name, builder);
             break;
         }
         case so::json_type::boolean: {
-            convert(value.get_bool().value(), field_name, builder);
+            convert(value.get_bool().value(), field_name, has_name, builder);
             break;
         }
         case so::json_type::null: {
-            convert_null(field_name, builder);
+            convert_null(field_name, has_name, builder);
             break;
         }
         }
         return Status::OK();
     }
 
-    static Status convert(SimdJsonObject obj, std::string_view field_name, vpack::Builder* builder) {
-        if (!field_name.empty()) {
+    static Status convert(SimdJsonObject obj, std::string_view field_name, bool has_name, vpack::Builder* builder) {
+        if (has_name) {
             builder->add(toStringRef(field_name), vpack::Value(vpack::ValueType::Object));
         } else {
             builder->add(vpack::Value(vpack::ValueType::Object));
         }
         for (auto field : obj) {
             std::string_view key = field.unescaped_key();
             auto value = field.value().value();
-            RETURN_IF_ERROR(convert(value, key, builder));
+            RETURN_IF_ERROR(convert(value, key, true, builder));
```

Comment:
How to determine passing true in this place?

Reply:
- true means object field, which has a field name
- false means array